code/dce-downloader/node/dcebulk.js

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env node
+
+const options = require('commander');
+const dce = require('./dce-common.js');
+const util = require('util');
+var errors = require('request-promise/errors');
```

Comment:
if we upgrade to Node 6 support use the request-promise-native library instead of request-promise. It will use the native Promise instead of a Promise library